iron-doc-function.html

```diff
@@ -18,23 +18,55 @@
   <template>
     <style include="iron-doc-viewer-styles"></style>
     <style>
+      #funcName {
+        color: #212121;
+      }
+
       .paramName {
-        margin-right: 10px;
+        color: var(--iron-doc-accent-color1);
       }
 
-      #params marked-element,
-      #params .markdown-html,
-      #params .markdown-html p {
+      .paramType, .returnType {
+        color: var(--iron-doc-accent-color2);
+      }
+
+      #paramList .paramName {
+        color: var(--iron-doc-accent-color2);
+        margin-right: 5px;
+      }
+
+      #paramList marked-element,
+      #paramList .markdown-html,
+      #paramList .markdown-html p {
         display: inline;
       }
     </style>
 
-    <div id="signature">
-      <a href$="#[[anchorId]]" class="name deeplink">[[descriptor.name]]</a>(<span class="params code"><span>[[_paramText]]</span></span>)<!--
-      --><span class="return" hidden$="[[!descriptor.return]]">: <span class="type code">[[descriptor.return.type]]</span></span>
-    </div>
+    <code id="signature">
+      <a href$="#[[anchorId]]" id="funcName" class="name deeplink">
+        [[descriptor.name]]<!--
+   --></a><!--
+   -->(<span class="funcParams"><!--
+     --><template is="dom-repeat" items="[[descriptor.params]]" as="p"><!--
+       --><span hidden$="[[!index]]">, </span><!--
+       --><span class="paramName">[[p.name]]</span><!--
+       --><span hidden$="[[!p.type]]">:
+            <span class="paramType">[[p.type]]</span><!--
+       --></span><!--
+     --></template><!--
+   --></span>)<!--
+   --><span hidden$="[[!descriptor.return]]">:
```

Comment:
This is bizzare. Let's try the `strip-whitespace` on `<template>` ([Ref](https://www.polymer-project.org/1.0/docs/devguide/local-dom#strip-whitespace))

Regardless, these should be auto collapsed already. Might be an issue since this is wrapped inside `<code>`. If that's the case, CSS `whitespace: normal` should fix this. ([Ref](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space?v=control))